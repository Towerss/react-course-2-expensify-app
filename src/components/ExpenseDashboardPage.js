import React from 'react';
import ExpensesSummary from './ExpensesSummary';
import ExpenseList from './ExpenseList';
import ExpenseListfilters from './ExpenseListFilters';

const ExpenseDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <ExpenseListfilters />
        <ExpenseList />
    </div>
);

export default ExpenseDashboardPage;