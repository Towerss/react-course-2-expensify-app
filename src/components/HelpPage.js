import React from 'react';

const HelpPage = () => (
    <div>
        This is from my HELP expenses component.
    </div>
);

export default HelpPage;