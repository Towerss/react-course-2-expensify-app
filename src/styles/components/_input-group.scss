.input-group{
    align-items: center;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    padding: $s-size;
    margin-bottom: $m-size;

    @media (min-width: $destop-breakpoint) {
        flex-direction: row;
        margin-bottom: $l-size;
    }
}

.input-group__item {
    margin-bottom: $s-size;
    @media (min-width: $destop-breakpoint) {
        margin: 0 $s-size 0 0;
    }
}