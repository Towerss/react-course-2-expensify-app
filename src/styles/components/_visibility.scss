.show-for-mobile {
 @media (min-width: $destop-breakpoint) {
     display: none;
 }
}

.show-for-desktop {
    @media (max-width: $destop-breakpoint - .01rem) {
        display: none;
    }
}