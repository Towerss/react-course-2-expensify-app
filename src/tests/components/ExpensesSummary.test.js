import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('Should render expenses summary with one expense and total', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={250} />);
    expect(wrapper).toMatchSnapshot();
});


test('Should render expenses summary with multiple expenses and total', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={23} expensesTotal={5461654} />);
    expect(wrapper).toMatchSnapshot();
});