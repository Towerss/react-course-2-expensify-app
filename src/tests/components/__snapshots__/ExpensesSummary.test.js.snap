// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render expenses summary with multiple expenses and total 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $54,616.54
  </h1>
</div>
`;

exports[`Should render expenses summary with one expense and total 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $2.50
  </h1>
</div>
`;
